EventFilter/HcalRawToDigi/plugins/HcalDigiToRawuHTR.cc

```diff
@@ -76,6 +78,8 @@ HcalDigiToRawuHTR::HcalDigiToRawuHTR(const edm::ParameterSet& iConfig)
       tok_HBHEDigiCollection_(consumes<HBHEDigiCollection>(iConfig.getParameter<edm::InputTag>("HBHEqie8"))),
       tok_HFDigiCollection_(consumes<HFDigiCollection>(iConfig.getParameter<edm::InputTag>("HFqie8"))),
       tok_TPDigiCollection_(consumes<HcalTrigPrimDigiCollection>(iConfig.getParameter<edm::InputTag>("TP"))),
+      tok_electronicsMap_(
+          esConsumes<HcalElectronicsMap, HcalElectronicsMapRcd>(edm::ESInputTag("", "electronicsMapLabel_"))),
```

Comment:
```suggestion
          esConsumes<HcalElectronicsMap, HcalElectronicsMapRcd>(edm::ESInputTag("", electronicsMapLabel_))),
```
The value is in the variable, not a string itself.